examples/example_plugin/example_plugin/jobs.py

```diff
@@ -44,4 +44,15 @@ def run(self, data, commit):
         return f"Ran for {interval} seconds"
 
 
-jobs = (ExampleJob, ExampleHiddenJob, ExampleLoggingJob)
+class ExampleJobHookReceiver(JobHookReceiver):
+    class Meta:
+        name = "Example job hook receiver"
+        description = "This job is triggered by object change events"
+
+    def run(self, data, commit):
+        change_action = data["object_change"].action
+        changed_object = data["object_change"].changed_object
+        self.log_success(message=f"{change_action} detected for object {changed_object.name}")
+
```

Comment:
Not blocking, but as I think about it, it seems that most non-trivial JobHookReceivers will want to see what has *changed* on the changed_object, not just what its final state is. For example, a Device-related JobHookReceiver might want to only do its thing when `device.status` is modified, and not when `device.comments` is modified.

Looking back at #1389, maybe it'd make sense to move `nautobot.extras.webhooks.get_snapshots()` and/or the body of `ObjectChangeView.get_extra_context()` to a utility module (or even to a method on the `ObjectChange` model) and provide an example of how the JobHookReceiver can use this function(s) to easily detect what has actually changed?

Another possibility might be to add a second ObjectVar to the JobHookReceiver class and have that populated with the *previous* ObjectChange, if any, leaving it up to the implementation to do the appropriate comparison between the two, but that feels less user-friendly to me.

Reply:
Added 3 new methods to `nautobot.extras.models.ChangeObject`:
- `get_related_changes(self)`
- `get_prev_change(self, user=None)`
- `get_next_change(self, user=None)`
- `get_snapshots(self)`